gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerViewModel.java

```diff
@@ -354,6 +440,26 @@ public void onCameraMove(CameraPosition newCameraPosition) {
             project -> projectRepository.setCameraPosition(project.getId(), newCameraPosition));
   }
 
+  public void onSavePolygonFeatureButtonClick(Nil nil) {
```

Comment:
Change no noargs `()` and call from binding.

Reply:
updated and moved to layout